--- build.sh.bak	2026-06-07 09:22:20
+++ build.sh	2026-06-07 09:22:13
@@ -461,7 +461,7 @@
 function build_dmg {
   if [[ -z ${MTX_VER} ]] fail Variable MTX_VER not set
 
-  if [[ -f packaging/macos/unlock_keychain.sh ]] packaging/macos/unlock_keychain.sh
+  if [[ -f ${SCRIPT_PATH}/unlock_keychain.sh ]] ${SCRIPT_PATH}/unlock_keychain.sh
 
   local dmgbase=${CMPL}/dmg-${MTX_VER}
   local dmgapp=$dmgbase/${APP_BUNDLE_NAME}
@@ -518,7 +518,7 @@
 EOF
 
   sed -e "s/::MTX_VER::/${MTX_VER}/" -e "s/::MACOSX_DEPLOYMENT_TARGET::/${MACOSX_DEPLOYMENT_TARGET}/" \
-      < packaging/macos/Info.plist \
+      < ${SCRIPT_PATH}/Info.plist \
       > $dmgcnt/Info.plist
 
   cat $dmgcnt/Info.plist
