--- build.sh.bak	2026-05-10 17:10:51
+++ build.sh	2026-05-10 17:06:20
@@ -411,17 +411,45 @@
   grep -q 'BUILD_GUI.*yes' build-config
 }
 
-function build_mkvtoolnix {
+function retrieve_verified_source_tarball {
   if [[ -z ${MTX_VER} ]] fail Variable MTX_VER not set
 
-  dmgbase=${CMPL}/dmg-${MTX_VER}
-  dmgcnt=$dmgbase/MKVToolNix-${MTX_VER}.app/Contents
-  dmgmac=$dmgcnt/MacOS
+  local public_key_name=${AUTHOR_PUBLIC_KEY_URL:t}
+  local tarball_name=mkvtoolnix-${MTX_VER}.tar.xz
+  local signature_name=${tarball_name}.sig
 
-  if [[ ! -f ${SRCDIR}/mkvtoolnix-${MTX_VER}.tar.xz ]]; then
-    curl -o ${SRCDIR}/mkvtoolnix-${MTX_VER}.tar.xz https://mkvtoolnix.download/sources/mkvtoolnix-${MTX_VER}.tar.xz
+  rm -f ${SRCDIR}/${public_key_name} ${SRCDIR}/${tarball_name} ${SRCDIR}/${signature_name}
+
+  curl -o ${SRCDIR}/${public_key_name} ${AUTHOR_PUBLIC_KEY_URL}
+  curl -o ${SRCDIR}/${tarball_name} ${SOURCES_URL}/${tarball_name}
+  curl -o ${SRCDIR}/${signature_name} ${SOURCES_URL}/${signature_name}
+
+  local keyring=$(uuidgen).keyring
+  local keybox=$( gpg --no-default-keyring --keyring ${keyring} \
+    --list-keys 2>&1 \
+    | awk -F"'" '/keybox.*created/ { print $2 }' )
+  if [[ -z ${keybox} ]]; then
+    fail Build requires an empty GPG keyring but {$keyring} already exists
   fi
 
+  gpg --no-default-keyring --keyring ${keyring} --import ${SRCDIR}/${public_key_name} 2>&1
+  
+  local signature_identity=$( gpg --no-default-keyring --keyring ${keyring} \
+    --verify ${SRCDIR}/${signature_name} ${SRCDIR}/${tarball_name} 2>&1 \
+    | awk -F"<|>" '/Good signature/ { print $2 }' )
+
+  if [[ ${signature_identity} != ${AUTHOR_IDENTITY} ]]; then
+    fail Source tarball ${tarball_name} is not signed by ${AUTHOR_IDENTITY}
+  fi
+
+  rm -f ${keybox}
+}
+
+function build_mkvtoolnix {
+  if [[ -z ${MTX_VER} ]] fail Variable MTX_VER not set
+
+  retrieve_verified_source_tarball
+
   NO_MAKE=1 NO_CONFIGURE=1 build_package /mkvtoolnix-${MTX_VER}.tar.xz
   build_configured_mkvtoolnix
 
