--- build.sh.bak	2026-05-11 13:17:04
+++ build.sh	2026-05-11 13:17:37
@@ -390,9 +390,9 @@
 function build_configured_mkvtoolnix {
   if [[ -z ${MTX_VER} ]] fail Variable MTX_VER not set
 
-  dmgbase=${CMPL}/dmg-${MTX_VER}
-  dmgcnt=$dmgbase/${APP_BUNDLE_NAME}/Contents
-  dmgmac=$dmgcnt/MacOS
+  local dmgbase=${CMPL}/dmg-${MTX_VER}
+  local dmgcnt=$dmgbase/${APP_BUNDLE_NAME}/Contents
+  local dmgmac=$dmgcnt/MacOS
 
   local -a args
   args=(
@@ -462,11 +462,11 @@
 
   if [[ -f packaging/macos/unlock_keychain.sh ]] packaging/macos/unlock_keychain.sh
 
-  dmgbase=${CMPL}/dmg-${MTX_VER}
-  dmgapp=$dmgbase/${APP_BUNDLE_NAME}
-  dmgcnt=$dmgapp/Contents
-  dmgmac=$dmgcnt/MacOS
-  latest_link=${CMPL}/latest
+  local dmgbase=${CMPL}/dmg-${MTX_VER}
+  local dmgapp=$dmgbase/${APP_BUNDLE_NAME}
+  local dmgcnt=$dmgapp/Contents
+  local dmgmac=$dmgcnt/MacOS
+  local latest_link=${CMPL}/latest
 
   rm -f ${latest_link}
 
@@ -549,7 +549,7 @@
       if [[ ${FILE} != */MacOS/mkv* ]] non_executables+=(${FILE})
     }
 
-    harden=""
+    local harden=""
     if [[ -n ${NOTARY_PROFILE} ]] harden="--options=runtime"
 
     codesign --force --sign ${SIGNATURE_IDENTITY} ${non_executables}
@@ -558,8 +558,8 @@
 
   if [[ -n $DMG_NO_DMG ]] return
 
-  machine=$(uname -m)
-  volumename=MKVToolNix-${MTX_VER}-${machine}
+  local machine=$(uname -m)
+  local volumename=MKVToolNix-${MTX_VER}-${machine}
   if [[ $DMG_PRE == 1 ]]; then
     # Ziel: 29.0.0-revision-008-gb71b2b27c-01808
     # describe: release-29.0.0-8-gb71b2b27c
@@ -572,15 +572,15 @@
     while [[ $build != *-*-*-* ]]; do
       build=${build}-0
     done
-    num=${${${build#release-}#*-}%-*}
-    hash=${build##*-}
-    revision="revision-$(printf '%03d' ${num})-${hash}-${build_number}"
+    local num=${${${build#release-}#*-}%-*}
+    local hash=${build##*-}
+    local revision="revision-$(printf '%03d' ${num})-${hash}-${build_number}"
 
     volumename=${volumename}-${revision}
   fi
 
-  dmgname=${CMPL}/MKVToolNix-${MTX_VER}-${machine}.dmg
-  dmgbuildname=${CMPL}/${volumename}.dmg
+  local dmgname=${CMPL}/MKVToolNix-${MTX_VER}-${machine}.dmg
+  local dmgbuildname=${CMPL}/${volumename}.dmg
 
   rm -f ${dmgname} ${dmgbuildname}
   hdiutil create -srcfolder ${dmgbase} -volname ${volumename} \
@@ -628,3 +628,4 @@
     shift;
   done
 fi
+
