--- build.sh.bak	2026-05-11 13:20:24
+++ build.sh	2026-05-11 13:00:33
@@ -558,8 +558,13 @@
 
   if [[ -n $DMG_NO_DMG ]] return
 
+  if [[ -z ${DMG_REVISION} ]]; then
+    fail Variable DMG_REVISION not set
+  fi
+
   local machine=$(uname -m)
-  local volumename=MKVToolNix-${MTX_VER}-${machine}
+  local dmg_label=MKVToolNix-${MTX_VER}-${DMG_REVISION}-${machine}
+  local volumename=${dmg_label}
   if [[ $DMG_PRE == 1 ]]; then
     # Ziel: 29.0.0-revision-008-gb71b2b27c-01808
     # describe: release-29.0.0-8-gb71b2b27c
@@ -579,7 +584,7 @@
     volumename=${volumename}-${revision}
   fi
 
-  local dmgname=${CMPL}/MKVToolNix-${MTX_VER}-${machine}.dmg
+  local dmgname=${CMPL}/${dmg_label}.dmg
   local dmgbuildname=${CMPL}/${volumename}.dmg
 
   rm -f ${dmgname} ${dmgbuildname}
@@ -628,4 +633,3 @@
     shift;
   done
 fi
-
