--- build.sh.bak	2026-05-11 13:00:33
+++ build.sh	2026-05-13 13:19:05
@@ -49,7 +49,7 @@
     fi
 
     echo "Warning: file ${file} exists but has the wrong checksum; retrieving anew"
-    rm ${file}
+    command rm ${file}
   fi
 
   if [[ ! -f ${file} ]]; then
@@ -290,31 +290,6 @@
     -DCMARK_TESTS=OFF \
     -DCMARK_STATIC=ON \
     -DCMARK_SHARED=OFF
-}
-
-function build_curl {
-  build_package curl \
-    --prefix=${TARGET} \
-    --disable-silent-rules \
-    --enable-ipv6 \
-    --without-brotli \
-    --without-cyassl \
-    --without-gnutls \
-    --without-gssapi \
-    --without-libmetalink \
-    --without-librtmp \
-    --without-libssh2 \
-    --without-nghttp2 \
-    --without-nss \
-    --without-polarssl \
-    --without-spnego \
-    --without-darwinssl \
-    --disable-ares \
-    --disable-ldap \
-    --disable-ldaps \
-    --with-zlib=${TARGET} \
-    --with-ssl=${TARGET} \
-    --with-ca-path=${TARGET}/etc/openssl/certs
 }
 
 function build_qt {
@@ -339,6 +314,7 @@
     -no-avx512
 
     -no-feature-cups
+    -no-feature-printsupport
     -no-feature-dbus
     -no-feature-glib
     -no-feature-openssl
@@ -418,23 +394,20 @@
   local tarball_name=mkvtoolnix-${MTX_VER}.tar.xz
   local signature_name=${tarball_name}.sig
 
-  rm -f ${SRCDIR}/${public_key_name} ${SRCDIR}/${tarball_name} ${SRCDIR}/${signature_name}
+  rm -f ${SRCDIR}/${public_key_name} ${SRCDIR}/${signature_name}
 
   curl -o ${SRCDIR}/${public_key_name} ${AUTHOR_PUBLIC_KEY_URL}
-  curl -o ${SRCDIR}/${tarball_name} ${SOURCES_URL}/${tarball_name}
   curl -o ${SRCDIR}/${signature_name} ${SOURCES_URL}/${signature_name}
 
-  local keyring=$(uuidgen).keyring
-  local keybox=$( gpg --no-default-keyring --keyring ${keyring} \
-    --list-keys 2>&1 \
-    | awk -F"'" '/keybox.*created/ { print $2 }' )
-  if [[ -z ${keybox} ]]; then
-    fail Build requires an empty GPG keyring but {$keyring} already exists
+  if [[ ! -f ${SRCDIR}/${tarball_name} ]]; then
+    curl -o ${SRCDIR}/${tarball_name} ${SOURCES_URL}/${tarball_name}
   fi
 
-  gpg --no-default-keyring --keyring ${keyring} --import ${SRCDIR}/${public_key_name} 2>&1
-  
-  local signature_identity=$( gpg --no-default-keyring --keyring ${keyring} \
+  local gpghome=$(mktemp -d)
+
+  gpg --homedir ${gpghome} --import ${SRCDIR}/${public_key_name} 2>&1
+
+  local signature_identity=$( gpg --homedir ${gpghome} \
     --verify ${SRCDIR}/${signature_name} ${SRCDIR}/${tarball_name} 2>&1 \
     | awk -F"<|>" '/Good signature/ { print $2 }' )
 
@@ -442,7 +415,7 @@
     fail Source tarball ${tarball_name} is not signed by ${AUTHOR_IDENTITY}
   fi
 
-  rm -f ${keybox}
+  rm -rf ${gpghome}
 }
 
 function build_mkvtoolnix {
@@ -478,7 +451,7 @@
 
   strip ${dmgcnt}/MacOS/mkv{merge,info,extract,propedit,toolnix-gui}
 
-  mv ${dmgmac}/mkvtoolnix ${dmgmac}/data
+  command mv ${dmgmac}/mkvtoolnix ${dmgmac}/data
 
   cp README.md $dmgbase/README.txt
   cp COPYING $dmgbase/COPYING.txt
@@ -524,7 +497,7 @@
 
   mkdir -p ${dmgmac}/libs
   cp -v -a \
-     ${TARGET}/lib/libQt6{Concurrent*.dylib,Core*.dylib,Gui*.dylib,Multimedia*.dylib,Network*.dylib,PrintSupport*.dylib,Svg*.dylib,Widgets*.dylib} \
+     ${TARGET}/lib/libQt6{Concurrent*.dylib,Core*.dylib,Gui*.dylib,Multimedia*.dylib,Network*.dylib,Svg*.dylib,Widgets*.dylib} \
      ${TARGET}/lib/libboost_system*.dylib \
      ${dmgmac}/libs/
 
@@ -598,7 +571,7 @@
     xcrun notarytool submit ${dmgname} --keychain-profile ${NOTARY_PROFILE} --wait
   fi
 
-  if [[ ${dmgname} != ${dmgbuildname} ]] mv ${dmgname} ${dmgbuildname}
+  if [[ ${dmgname} != ${dmgbuildname} ]] command mv ${dmgname} ${dmgbuildname}
 
   ln -s ${dmgbuildname} ${latest_link}
 }
