--- specs.sh.bak	2026-05-10 10:53:06
+++ specs.sh	2026-05-13 13:19:05
@@ -3,7 +3,6 @@
 spec_boost=(boost_1_88_0.tar.bz2 https://archives.boost.io/release/1.88.0/source/boost_1_88_0.tar.bz2 46d9d2c06637b219270877c9e16155cbd015b6dc84349af064c088e9b5b12f7b)
 spec_cmake=(cmake-3.31.3.tar.gz https://cmake.org/files/v3.31/cmake-3.31.3.tar.gz fac45bc6d410b49b3113ab866074888d6c9e9dc81a141874446eb239ac38cb87)
 spec_cmark=(cmark-0.30.3.tar.gz https://github.com/commonmark/cmark/archive/0.30.3.tar.gz 85e9fb515531cc2c9ae176d693f9871774830cf1f323a6758fb187a5148d7b16)
-spec_curl=(curl-8.11.1.tar.xz https://curl.se/download/curl-8.11.1.tar.xz c7ca7db48b0909743eaef34250da02c19bc61d4f1dcedd6603f109409536ab56)
 spec_docbook_xsl=(docbook-xsl-1.79.2.tar.gz https://github.com/docbook/xslt10-stylesheets/releases/download/release%2F1.79.2/docbook-xsl-1.79.2.tar.gz 966188d7c05fc76eaca115a55893e643dd01a3486f6368733c9ad974fcee7a26)
 spec_flac=(flac-1.5.0.tar.xz https://downloads.xiph.org/releases/flac/flac-1.5.0.tar.xz f2c1c76592a82ffff8413ba3c4a1299b6c7ab06c734dee03fd88630485c2b920)
 spec_gettext=(gettext-0.23.tar.lz https://ftp.gnu.org/gnu/gettext/gettext-0.23.tar.lz 9878994930a8e09f1e3186203b3ff2accb2c4e67ddc6098b2c1e4a74364d3c5a)
