--- specs.sh.bak	2026-05-13 13:34:26
+++ specs.sh	2026-05-13 13:34:34
@@ -12,5 +12,5 @@
 spec_pkgconfig=(pkg-config-0.29.2.tar.gz https://pkg-config.freedesktop.org/releases/pkg-config-0.29.2.tar.gz 6fc69c01688c9458a57eb9a1664c9aba372ccda420a02bf4429fe610e7e7d591)
 spec_vorbis=(libvorbis-1.3.7.tar.xz http://downloads.xiph.org/releases/vorbis/libvorbis-1.3.7.tar.xz b33cc4934322bcbf6efcbacf49e3ca01aadbea4114ec9589d1b1e9d20f72954b)
 spec_zlib=(zlib-v1.3.2.tar.xz https://zlib.net/zlib-1.3.2.tar.xz d7a0654783a4da529d1bb793b7ad9c3318020af77667bcae35f95d0e42a792f3)
-
 spec_qt=(qt-everywhere-src-6.11.0.tar.xz https://download.qt.io/archive/qt/6.11/6.11.0/single/qt-everywhere-src-6.11.0.tar.xz acf3b3db04c9e5d0820e8324b097320388954c297cee83d2bd698789234f68a4)
+spec_gpg=(gnupg-1.4.23.tar.bz2 https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-1.4.23.tar.bz2 c9462f17e651b6507848c08c430c791287cd75491f8b5a8b50c6ed46b12678ba)
