--- specs.sh.bak	2026-05-13 13:49:51
+++ specs.sh	2026-05-13 13:51:50
@@ -14,3 +14,5 @@
 spec_zlib=(zlib-v1.3.2.tar.xz https://zlib.net/zlib-1.3.2.tar.xz d7a0654783a4da529d1bb793b7ad9c3318020af77667bcae35f95d0e42a792f3)
 
 spec_qt=(qt-everywhere-src-6.10.0.tar.xz https://download.qt.io/official_releases/qt/6.10/6.10.0/single/qt-everywhere-src-6.10.0.tar.xz 81895fb038a9c3d6c6f698d7611339a189eb45c3d91746c7789b0b77a5981aa3)
+spec_gpg=(gnupg-1.4.23.tar.bz2 https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-1.4.23.tar.bz2 c9462f17e651b6507848c08c430c791287cd75491f8b5a8b50c6ed46b12678ba)
+
