--- build.sh.bak	2026-05-13 16:27:37
+++ build.sh	2026-05-13 16:27:47
@@ -363,6 +363,10 @@
   ln -s ${${spec_docbook_xsl[1]}%%.tar*} ${DOCBOOK_XSL_ROOT_DIR}
 }
 
+function build_gpg {
+  build_package gpg --prefix=${TARGET}
+}
+
 function build_configured_mkvtoolnix {
   if [[ -z ${MTX_VER} ]] fail Variable MTX_VER not set
 
@@ -598,6 +602,7 @@
   build_gmp
   build_boost
   build_qt
+  build_gpg
   build_mkvtoolnix
 
 else
